LazerPay-Finance/Lazerpay-documentation-v2
# Verify payment

#### What does this Merge request do?
- Verify payment page

#### Description of Task to be completed?
Pr to create verify payment page

#### How should this be manually tested?
- Pull this branch.
- Navigate to verify payment page

#### Checks

- [x] tested changes after development?
- [x] Cleaned up all console.log & console warnings?

#### Any background context you want to provide?

xxx

#### Screenshots (if appropriate):

xxx

#### Questions (if any):

xxx
